var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -13,25 +13,45 @@ class Fms(CMakePackage):
     system models."""
 
     homepage = "https://github.com/NOAA-GFDL/FMS"
-    url      = "https://github.com/NOAA-GFDL/FMS/archive/refs/tags/2021.02.01.tar.gz"
-
-    maintainers = ['kgerheiser', 'Hang-Lei-NOAA', 'edwardhartnett']
-
-    variant('64bit', default=True, description='64 bit?')
-    variant('gfs_phys', default=True, description='Use GFS Physics?')
-    variant('openmp', default=True, description='Use OpenMP?')
-
-    version('2021.04', sha256='dcb4fe80cb3b7846f7cf89b812afff09a78a10261ea048a851f28935d6b241b1')
+    url      = "https://github.com/NOAA-GFDL/FMS/archive/refs/tags/2022.02.tar.gz"
+    git      = "https://github.com/NOAA-GFDL/FMS.git"
+
+    maintainers = ['kgerheiser', 'Hang-Lei-NOAA', 'edwardhartnett', 'rem1776']
+
+    variant('64bit', default=True, description='Build a version of the library with default 64 bit reals')
+    variant('gfs_phys', default=True, description='Use GFS Physics')
+    variant('openmp', default=True, description='Use OpenMP')
+    variant('quad_precision', default=True, description='quad precision reals')
+    variant('yaml', default=False, description='yaml input file support(requires libyaml)', when='@2021.04')
+    variant('constants', default='GFDL', description='Build with <X> constants parameter definitions',
+            values=('GFDL','GEOS','GFS'), multi=False, when='@2022.02')
```

Comment:
Maybe you meant:
```suggestion
            values=('GFDL','GEOS','GFS'), multi=False, when='@2022.02:')
```
? (The option will not disappear after 2022.02)

Reply:
I had thought specifying the version this way automatically worked with any more recent versions, does it need the `:` as well?

Reply:
Yes

Reply:
Thanks for pointing that out, all the comments should be addressed now